apps/nextjs/src/app/home-page.tsx

```diff
@@ -162,26 +160,19 @@ export default function HomePage({ featureFlag }) {
                 resources.
               </OakP>
 
-              {!userId || !featureFlag ? (
```

Comment:
This looks like a functional change if the userId doesn't immediately load. I don't know if that can happen

Reply:
This was to do with creating a button saying coming soon or not weather the user was logged in